src/main/java/controller/RootController.java

```diff
@@ -0,0 +1,13 @@
+package controller;
+
+import http.*;
+
+public class RootController extends Controller {
+
+    protected void doGet(HttpRequest request, HttpResponse response) {
+        byte[] body = "Hello World".getBytes();
+
+        response.setContentType(ContentType.of("html"));
```

Comment:
`ContetntType.HTML` 상수를 정의해뒀으니 이 상수를 활용하는 것이 조금 더 좋을 것 같아요. 🙂

```suggestion
        response.setContentType(ContentType.HTML);
```